templates/upcoming.html

```diff
@@ -14,17 +14,17 @@ <h3>Release timeline</h3>
     </div>
     <!-- <paper-toggle-button> doesn't working here. Related links:
       https://github.com/PolymerElements/paper-toggle-button/pull/132 -->
-    <label><input type="checkbox" class="show-blink-checkbox">Show Blink components</label>
+    <label><input type="checkbox" class="show-blink-checkbox">Show Shipping Type</label>
```

Comment:
Show Shipping Type replaces the Show Blink Components button. If you approve the UI, we can make the change to show the shipping types by default in a later PR.

Reply:
OK.